### PR TITLE
CP-927: Resource Owner Template

### PR DESCRIPTION
**Pull request by Tankanow**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,7 +47,9 @@ lint: lint-templates lint-sam-apps
 .PHONY: lint-templates
 lint-templates: $(CFN_LINT_OUTPUT)
 $(CFN_LINT_OUTPUT): $(CFN_TEMPLATES)
-	cfn-lint -t $(CFN_TEMPLATES)
+	for t in $(CFN_TEMPLATES) ; do \
+		cfn-lint -t $${t} ; \
```
Comment (killthrush) [inline Makefile]: We should use `cfn-lint` everywhere IMO, really useful 👍

File: services/account_type/resource_owner.yaml [hunk 2]
```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016-present, CloudZero, Inc. All rights reserved.
+# Licensed under the BSD-style license. See LICENSE file in the project root for full license information.
+
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CloudZero Connected Account Template
```
Comment (killthrush) [inline services/account_type/resource_owner.yaml]: Update description?

File: services/account_type/resource_owner.yaml [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+          - iam:Simulate*
+          - lambda:List*
+          - lambda:GetAccountSettings
+          - lambda:GetAlias
+          - lambda:GetEventSourceMapping
+          - lambda:GetFunctionConfiguration
+          - lambda:GetPolicy
+          - machinelearning:Describe*
+          - opsworks:Describe*
+          - opsworks-cm:Describe*
```
Comment (killthrush) [inline services/account_type/resource_owner.yaml]: FWIW I have a ticket entered to get you guys some more information to update this list.  I got pre-empted to work on other things though so I'm not likely to get there anytime soon

File: services/account_type/resource_owner.yaml (hunk 2 above)
Comment (Tankanow) [reply services/account_type/resource_owner.yaml]: good catch; done

**Comment by Tankanow** [comment]
OK @killthrush, I think this is ready for re-review

**Comment by jeremyhahn** [comment]
:+1:

**Pull request merged**